raiden-dapp/src/services/raiden-service.ts

```diff
@@ -71,6 +71,9 @@ export default class RaidenService {
           ...(process.env.VUE_APP_SETTLE_TIMEOUT && +process.env.VUE_APP_SETTLE_TIMEOUT
             ? { settleTimeout: +process.env.VUE_APP_SETTLE_TIMEOUT }
             : undefined),
+          ...(process.env.VUE_APP_EXPIRY_FACTOR && +process.env.VUE_APP_EXPIRY_FACTOR
+            ? { expiryFactor: +process.env.VUE_APP_EXPIRY_FACTOR }
+            : { expiryFactor: 1.1 }),
```

Comment:
@andrevmatos What's the best way to add validation here? I'd liek to make sure this value is >= 1.1

Reply:
You would need a custom codec for this. I'm not sure it's worth it, since this is input by the developer.
If you wanna do it anyway, you would need the `expiryFactor` in `const RaidenConfig` to be a variable declared (inline or separete) as a `t.Type<number, number>`, `pipe`ing `t.number.decode` into a `fold` with your custom validator (e.g. `(u) => u > 1`). Check `BigNumberC` in `src/utils/types.ts` and `io-ts` docs.

Reply:
Yeah, the improvement doesn't seem to be worth the added complexity.